ethcore/src/engines/authority_round/mod.rs

```diff
@@ -79,6 +81,7 @@ impl From<ethjson::spec::AuthorityRoundParams> for AuthorityRoundParams {
 			validate_step_transition: p.validate_step_transition.map_or(0, Into::into),
 			immediate_transitions: p.immediate_transitions.unwrap_or(false),
 			block_reward: p.block_reward.map_or_else(Default::default, Into::into),
+			maximum_uncle_count_transition: p.maximum_uncle_count_transition.map_or(0, Into::into),
```

Comment:
This will silently break existing networks for Aura users. Consider setting to `u64::max_value()` instead.

Reply:
It's keeping the behavour introduced in #7006, I think we don't really want anyone running AuRa with uncles. @keorn ?